api/monitor.py

```diff
@@ -72,7 +73,16 @@ def where_clause(self):
         very_old_with_no_clear_end_date = and_(
             start_field < self.cutoff, end_field == None
         )
-        return and_(superclause, or_(expired, very_old_with_no_clear_end_date))
+        not_unlimited_access = LicensePool.unlimited_access == False
+        return and_(
+            superclause,
+            not_unlimited_access,
+            or_(expired, very_old_with_no_clear_end_date),
+        )
+
+    def query(self):
```

Comment:
For what is this `query` method used?

Reply:
The query() method is an overridden method from `core.monitor.ReaperMonitor`. We override it here to add the additional `join(licensepool)` to the SQL.